composer.json

```diff
@@ -7,7 +7,7 @@
     "license": "BSD-3-Clause",
     "require": {
         "php": ">=7.4",
-        "google/auth": "^1.25.0",
+        "google/auth": "dev-add-universe-domain-phase2 as 1.31.0",
```

Comment:
This is here until we merge https://github.com/googleapis/google-auth-library-php/pull/477, at which point we need to update it to the corresponding auth library release